IFC4x3/Properties/w/WorkingState_2qD57tYLb86QIkv8vmjv3f/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicates the working state of the Anemometrograph.
```

Comment:
Very generic name for a quite specific property description. Could an example be provided since this is an IfcLabel? What could be the working state?